doc/config.example.yaml

```diff
@@ -135,7 +135,10 @@ flash:
   # Maximum number of seconds to re-try a failed payment / update request
   # before the request is considered failed and is reported to the listener
   # (wallet / etc)
-  max_retry_time: 60
+  max_retry_time: 60000
```

Comment:
Why ?

Reply:
because it is in milliseconds, and agora would crash without that change
```D
enforce(max_retry_time > max_retry_delay, "`max_retry_time` must be greater than `max_retry_delay`");
```

Reply:
It should be in seconds, as the comment says.

Reply:
```D
    const max_retry_time = get!(Duration, "flash", "max_retry_time", str => str.to!ulong.msecs)
```